docs/changelog.md

```diff
@@ -13,114 +13,170 @@ sidebar_label: Changelog
 * Allow duration values to be used as weights in the [Weighted Shortest
   Path](https://memgraph.com/docs/memgraph/reference-guide/graph-algorithms#weighted-shortest-path)
   query. [#278](https://github.com/memgraph/memgraph/pull/278)
-* Fix linkage error when `mgp_local_time_get_minute` is used. [#273](https://github.com/memgraph/memgraph/pull/273)
+* Fix linkage error when `mgp_local_time_get_minute` is used.
+  [#273](https://github.com/memgraph/memgraph/pull/273)
+* Added Bolt notifications to inform the user about results or to give useful
+  tips. [#285](https://github.com/memgraph/memgraph/pull/285)
+* Added execution statistics to inform user on how much object were affected.
```

Comment:
It should be either:
-"how many objects were affected" or
-"how objects were affected".

Reply:
its the first